iree/jax/frontend.py

```diff
@@ -25,12 +25,12 @@
 
 try:
   import jax
-except ModuleNotFoundError as e:
-  raise ModuleNotFoundError("iree.jax requires 'jax' and 'jaxlib' to be "
+except ProgramNotFoundError as e:
+  raise ProgramNotFoundError("iree.jax requires 'jax' and 'jaxlib' to be "
                             "installed in your python environment.") from e
 
 # pytype thinks iree.jax is jax.
-# pytype: disable=module-attr
+# pytype: disable=program-attr
```

Comment:
I think these should be left as is.

Reply:
Thanks, changed back to use Module / module